src/extension/ext-raw.js

```diff
@@ -11,7 +11,7 @@ const navigationOptions = [
   { value: 'expandAll', translation: 'Object.OrgChart.ExpandAll' },
 ];
 
-const bordersActive = data =>
+const bordersActive = (data) =>
```

Comment:
my linter does this as well, do we want it with brackets??

Reply:
I just go with what the linter does. It is updated every week I think, so probably this is coming from one of those lint updates